hmt_escrow/job.py

```diff
@@ -121,69 +121,49 @@ def manifest_hash(escrow_contract: Contract,
     })
 
 
-def intermediate_url(escrow_contract: Contract,
+def recording_oracle_url(escrow_contract: Contract,
                      gas_payer: str,
                      gas: int = GAS_LIMIT) -> str:
-    """Retrieves the deployed intermediate results url uploaded on Job initialization.
+    """Retrieves the deployed recording oracle results url uploaded on Job initialization.
 
     Args:
         escrow_contract (Contract): the escrow contract of the Job.
         gas_payer (str): an ethereum address paying for the gas costs.
         gas (int): maximum amount of gas the caller is ready to pay.
 
     Returns:
-        str: returns the intermediate results url of Job's escrow contract.
+        str: returns the recording oracle results url of Job's escrow contract.
 
     """
-    return escrow_contract.functions.getIntermediateResultsUrl().call({
+    return escrow_contract.functions.getRecordingOracleResultsUrl().call({
         'from':
         gas_payer,
         'gas':
         gas
     })
 
 
-def intermediate_hash(escrow_contract: Contract,
+def recording_oracle_hash(escrow_contract: Contract,
                       gas_payer: str,
                       gas: int = GAS_LIMIT) -> str:
-    """Retrieves the deployed intermediate results hash uploaded on Job initialization.
+    """Retrieves the deployed recording oracle results hash uploaded on Job initialization.
 
     Args:
         escrow_contract (Contract): the escrow contract of the Job.
         gas_payer (str): an ethereum address paying for the gas costs.
         gas (int): maximum amount of gas the caller is ready to pay.
 
     Returns:
-        str: returns the intermediate results hash of Job's escrow contract.
+        str: returns the recording oracle results hash of Job's escrow contract.
 
     """
-    return escrow_contract.functions.getIntermediateResultsHash().call({
+    return escrow_contract.functions.getRecordingOracleResultsHash().call({
         'from':
         gas_payer,
         'gas':
         gas
     })
 
-
-def launcher(escrow_contract: Contract, gas_payer: str,
```

Comment:
This probably shouldn't be deleted, since we use it to whitelist in another service.

Reply:
k, it is currently in!
https://github.com/hCaptcha/hmt-escrow/blob/ipns/hmt_escrow/job.py#L857